contracts/src/deployers/erc4626/ERC4626HyperdriveDeployerCoordinator.sol

```diff
@@ -18,6 +18,10 @@ import { HyperdriveDeployerCoordinator } from "../HyperdriveDeployerCoordinator.
 contract ERC4626HyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
     using SafeERC20 for ERC20;
 
+    /// @notice The deployer coordinator's name.
+    string public constant override name =
```

Comment:
should this be in caps?

Reply:
Since it's public, we can leave it as lower-case